lib/openai.rb

```diff
@@ -52,4 +52,14 @@ def self.configuration
   def self.configure
     yield(configuration)
   end
+
+  def self.rough_token_count(content = "")
```

Comment:
Can you add your explanation from the PR description here

```suggestion

  # Explain what this does
  # Link to OpenAI docs
  def self.rough_token_count(content = "")
```

Reply:
Done, in 3845a45a8664b6b0c5f0096d303622ca017e58bd.

If [that new section in the README](https://github.com/alexrudall/ruby-openai/blob/3845a45a8664b6b0c5f0096d303622ca017e58bd/README.md#counting-tokens) is too wordy, let me know and I'll cut it down.